packages/api-mobx/package.json

```diff
@@ -20,5 +20,8 @@
   },
   "peerDependencies": {
     "@polkadot/api": "4.*"
+  },
+  "devDependencies": {
+    "@acala-network/api": "^0.7.4-20"
```

Comment:
why not latest version?

Reply:
This is used for testing. I think this is the latest one. Let me check